libs/ledger-live-common/src/families/cardano/js-estimateMaxSpendable.ts

```diff
@@ -42,6 +43,7 @@ const estimateMaxSpendable = async ({
   try {
     typhonTransaction = await buildTransaction(a as CardanoAccount, t);
   } catch (error) {
+    log("cardano-error", "Failed to estimate max spendable: " + String(error));
     return new BigNumber(0);
```

Comment:
Can we even continue the normal flow if we get in the catch clause ?

Reply:
indeed, i wonder if it's necessary to keep this try-catch part. basically my `(2)` question in description for the cardano maintainers 🙏 
but if we keep it it must be both documented with a comment and we need to log the error as suggested here ;)

Reply:
Agree with you... should we create a ticket to try to remove the try catch thingie ? @haammar-ledger any thoughts ?

Reply:
Maybe we can first log the thing and then by this ticket try to investigate if we can remove the try catch safely